src/models/AssetFormatter.ts

```diff
@@ -0,0 +1,94 @@
+import Decimal from "decimal.js";
+import * as Long from "long";
+import { DCoreConstants } from "../DCoreConstants";
+import { AssetPrecision } from "../DCoreSdk";
+import { Asset } from "./Asset";
+import { AssetAmount } from "./AssetAmount";
+import { ChainObject } from "./ChainObject";
+
+export class AssetFormatter {
```

Comment:
Shouldn't asset somehow implement this? How are the methods accessible in Asset class?